02-peer-protocol.md

```diff
@@ -403,14 +396,13 @@ The receiving node:
   - SHOULD apply the `witness`es to the transaction and broadcast it
   - MUST reply with their `tx_signatures` if not already transmitted
 
-
 #### Rationale
+
 The peer with the lowest total of inputs amounts must transmit its
 `tx_signatures` first. This gives in a strict ordering of transmission
 for any multiparty tx collaboration.
 
 `witness` is the data for a witness element in a witness stack.
-Witness elements should *not* include their length.
```

Comment:
@t-bast really out to get the length havers haha.

Reply:
Haha I don't really remember why I removed that...maybe because I didn't see why we would include their length, so it was confusing when I implemented it? Can you clarify what made you add this?

Reply:
Typically, in the witness fields of a transaction, you'd include the length of the witness element, then the element itself. This is just clarifying that those length counts should be omitted.

Reply:
Ok, it seemed obvious to me since we're using tlv and already explicitly include a length right before, but we can keep that if you think it's helpful, I'll restore it!